### DISCUSSION TITLE
Coolify CDN inaccessible from Iran - Provide a fallback way via GH

### DISCUSSION THREAD

**Discussion by amirparsadd** [coollabsio/coolify#4652] [Improvement Requests]
### Error Message and Logs

so if you try to install coolify on ubuntu when you are in iran, you are unable to access the cdn. is there any workaround? like building coolify ourselves on the server?

### Steps to Reproduce

1. Live in iran
2. Try to download coolify on ubuntu 24.x without a dns server
3. Wait 5 minutes

### Coolify Version

latest

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 24.x

**Comment by peaklabs-dev** [comment]
Actually, there is a better way. You can use the script directly from GitHub instead of the CDN:
`curl -fsSL https://raw.githubusercontent.com/coollabsio/coolify/main/scripts/install.sh | sudo bash`. 

Does this work for you? If it does, I will add it to the docs.

**Edit: Never mind in the install script also needs to access the CDN again, so does the manual installation** 

-> We need to change the install script and the manual install to have a fallback way via GitHub, since both currently use the CDN.

**Comment by amirparsadd** [comment]
But then it's still using the cdn for other stuff so the best solution i
have found is using a dns like "Shecan" for persian users. I'm still not
sure how to use a dns in linux, i just edit the /etc/resolv.conf but it
just gets reverted every now and then.
